test/OpenTelemetry.Api.Tests/Trace/TracerTest.cs

```diff
@@ -62,13 +62,13 @@ public void Tracer_StartRootSpan_BadArgs_NullSpanName()
             .Build();
 
         var span1 = this.tracer.StartRootSpan(null);
-        Assert.Null(span1.Activity.DisplayName);
+        Assert.True(string.IsNullOrEmpty(span1.Activity.DisplayName));
```

Comment:
Why making this change?

Reply:
Probably this: https://github.com/dotnet/runtime/pull/77746